pretix_zugferd/templates/pretix_zugferd/settings.html

```diff
@@ -270,6 +270,24 @@ <h2>{% trans "Configuration check" %}</h2>
                 {% endif %}
             </td>
         </tr>
+        <tr>
+            <td>{% trans "Rounding" %}</td>
+            <td>
+                {% if "sum_by_net" not in request.event.settings.tax_rounding %}
+                    <span class="fa fa-fw fa-warning text-danger"></span>
+                    {% trans "Taxes are rounded by line by line." %}
```

Comment:
```suggestion
                    {% trans "Taxes are rounded line by line." %}
```
Suggestion based on language comprehension, if the original makes more sense in context, please ignore